src/xtgeo/xyz/_polygons_oper.py

```diff
@@ -135,17 +139,17 @@ def _create_boundary_polygon(
             "Your alpha or alpha_factor value is too low, try increasing it!"
         )
 
-    polygons = _get_shapely_polygons_from_edges(coords, edges)
+    pol_coords = _get_polygon_coords_from_edges_shapely(coords, edges)
 
     # sort polygons by their size to get the largest polygons first
-    sorted(polygons, key=lambda pol: len(pol.exterior.coords), reverse=True)
+    sorted(pol_coords, key=len, reverse=True)
```

Comment:
This cant be right?

Reply:
Thought it was correct, this pol_coords is now a list of lists with coordinates, and I want the one with most coordinates (largest polygon) to be first in the list. But I'll take an extra check 🙂

Reply:
I get that, but this is not an inplace sort. 

```suggestion
    pol_coords = sorted(pol_coords, key=len, reverse=True)
```

Reply:
Nice catch the list is not sorted in place!

Reply:
Lists has a .sort method that does in place sorting.

ex.
```python
x = [3,4,1,2,2]
x.sort()
```